### PR TITLE
MCD migration - Add reference to Azure migration guidance

### PR DESCRIPTION
**Pull request by heikobischoff**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md [hunk 1]
```diff
@@ -28,12 +28,15 @@ Tenant migrations from Microsoft Cloud Deutschland (MCD) to the region "Germany"
 
 The migration process will complete over many weeks depending on the overall size and complexity of the organization. While the migration is underway, users and administrators are able to continue utilizing the services with notable changes detailed in this documentation. The graphic and table define phases and steps during the migration.
 
+> [!NOTE]
+> The migration of Azure services is not part of this documentation. Please refer to the [Migration guidance for Azure Germany](https://docs.microsoft.com/en-us/azure/germany/germany-migration-main).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-phases.md]:
```suggestion
> The migration of Azure services is not part of this documentation. Please refer to the [Migration guidance for Azure Germany](/azure/germany/germany-migration-main).
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md [hunk 2]
```diff
@@ -28,12 +28,15 @@ Tenant migrations from Microsoft Cloud Deutschland (MCD) to the region "Germany"
...
+
 |Step|Duration|Responsible party|Description|
 |:--------|:--------|:--------|:--------|
 |Opt-In|Hours|Customer|Opt your organization into the migration.|
 |Pre-Work|Days|Customer|Complete the work needed to prepare users, workstations, and network for migration.|
 |Azure Active Directory (Azure AD)|1-2 days|Microsoft|Migrate Azure AD organization to worldwide.|
-|Azure|Weeks|Customer|Create new worldwide Azure subscriptions and transition Azure services.|
+|Azure|Weeks|Customer|Create new worldwide Azure subscriptions and [transition Azure services](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/move-resource-group-and-subscription).|
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-phases.md]:
```suggestion
|Azure|Weeks|Customer|Create new worldwide Azure subscriptions and [transition Azure services](/azure/azure-resource-manager/management/move-resource-group-and-subscription).|
```

File: microsoft-365/enterprise/ms-cloud-germany-transition.md [hunk 3]
```diff
@@ -97,6 +96,9 @@ Existing Microsoft Cloud Deutschland customers can now begin to migrate their Of
 
 As a result of the migration, core customer data and subscriptions are moved to the new German datacenter regions.
 
+> [!NOTE]
+> This documentation comprises guidance for the migration of Office 365 services only. Customers which are running additional Azure workloads in Microsoft Cloud Deutschland must consider the [Migration guidance for Azure Germany](https://docs.microsoft.com/en-us/azure/germany/germany-migration-main).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition.md]:
```suggestion
> This documentation comprises guidance for the migration of Office 365 services only. Customers who are running additional Azure workloads in Microsoft Cloud Deutschland must consider the [Migration guidance for Azure Germany](/azure/germany/germany-migration-main).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @heikobischoff @yogkumgit @JohanFreelancer9

**Pull request merged**